var/spack/repos/builtin/packages/xmlto/package.py

```diff
@@ -19,3 +19,4 @@ class Xmlto(AutotoolsPackage):
 
     # FIXME: missing a lot of dependencies
     depends_on("libxslt")
+    depends_on("util-linux", when="platform=darwin")  # macOS getopt does not support longopts
```

Comment:
`xmlto` is a shell script, and the installed script uses `getopt`, so the dependency should probably be `type=("build", "run")`.

Also, on my Arch linux system the installed `getopt` comes from the `util-linux` package. So the dependency is the same on linux, it's probably just virtually always satisfied by the distro. Perhaps `util-linux` should be added as a dependency independent from the platform?